src/core/sayt.ts

```diff
@@ -98,6 +98,42 @@ export interface ProductSearchConfig {
   productSort?: any;
 }
 
+export interface AutocompleteResponse {
+  status: {
+    code: number;
+    internalCode: number;
+    message: 'string';
+    additionalInfo: any;
+    serverTimeStamp: number;
+  }
+  result: {
+    stats: {
+      navigationCount: number;
+      searchCount: number;
+      productCount: number;
+      autocompleteResponse: number;
+      productSearchResponse: number;
+    }
+    searchTerms: AutocompleteSearchTerm[] | null;
+    navigations: AutocompleteNavigation[] | null;
+    products: any;
```

Comment:
This is probably going to always be an array so let's do:
```suggestion
    products: any[];
```
or if we want to account for a more standardized model in the future we can create a Product(data?) interface that is just typed `any` for now

Reply:
Have added an interface for products and set it to 
```
products: AutocompleteProducts[] | null
```